### PR TITLE
Aggregate calls

### PR DESCRIPTION
**Pull request by t-animal**
Meh. How about this RxJS?

### CODE REVIEW

File: frontend-mobile/lib/view/cubit/coffee_call_cubit.dart [hunk 1]
```diff
@@ -38,7 +38,7 @@ class CoffeeCallCubit extends Cubit<CoffeeCallState> {
     protocol.messages.listen((message) {
       emit(state.copyWith(messages: [
         ...state.messages,
-        '${message.name} ${message.type.name}s a coffee call',
+        '${message.name} ${message.type.name}s a coffee call @${message.broadcastAt}',
```
Comment (mxwnk) [inline frontend-mobile/lib/view/cubit/coffee_call_cubit.dart]: Maybe something like: `${message.broadcastAt.hour.toString().padLeft(2,'0')}:${now.minute.toString().padLeft(2,'0')}`
Or as extension method: `broadcastAt.toTimeString()`

File: frontend-web/src/coffee-call/coffee-call.tsx [hunk 2]
```diff
@@ -17,8 +21,14 @@ export const CoffeeCall: Component = () => {
...
-        <For each={protocol.messages()}>{(message) =>
-          <li>{message.name} {message.type}s a coffee call</li>
+        <Show when={call().state === CoffeeCallState.announced}>Coffee call ausgerufen!</Show>
+        <Show when={call().state === CoffeeCallState.started}>Coffee call gestartet!</Show>
+        <Show when={call().state === CoffeeCallState.canceled}>Coffee call abgebrochen!</Show>
+
+        Mit dabei: {participantsAsList()}
+
+        <For each={call().messages}>{(message) =>
+          <li>{message.name} {message.type}s a coffee call @{message.broadcastAt.toLocaleString()}</li>
```
Comment (mxwnk) [inline frontend-web/src/coffee-call/coffee-call.tsx:46]: Are we switching to denglisch now?

**Review by mxwnk — APPROVED** [review]
LGTM :)

**Comment by t-animal** [comment]
Completely overhauled this. Will close and re-open for clarity.

**Pull request closed without merging**